### PR TITLE
tar is bundled with MacOS. Fix #38

### LINKED ISSUE #38: Add channel support to Snap
Would like to be able to install packages in Snap other than stable. Is this possible?

### PR DESCRIPTION
**Pull request by noelmcloughlin**
This PR fixes issue #38.  Tar is always bundled with MacOS so no homebrew package exists.

### CODE REVIEW

**Comment by noelmcloughlin** [comment]
tar is preinstalled on GNU/Linux. http://www.gnu.org/software/tar/manual/tar.html

File: packages/archives.sls [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # vim: ft=sls
 {% from "packages/map.jinja" import packages with context %}
 
-{% set req_packages = packages.pkgs.required.pkgs + ['curl', 'tar', 'bzip2', 'gzip',] %}
+{% set req_packages = packages.pkgs.required.pkgs + ['curl', 'bzip2', 'gzip',] %}
```
Comment (javierbertoli) [inline packages/archives.sls]: How about moving these to the `defaults.yaml`, `osfamilymap.yaml`, etc files? they'll be merged my `map.jinja`, right?

**Comment by noelmcloughlin** [comment]
Brew packages exist for `'curl', 'bzip2', 'gzip',` so these are okay.

File: packages/archives.sls (hunk 1 above)
Comment (noelmcloughlin) [reply packages/archives.sls]: Sure thats best idea. thx

**Comment by noelmcloughlin** [comment]
I'm surprise python cannot merge tuple (a, b) into list [c, d] - surely the intention is clear .. keep the list, lose the tuple.  Anyway, I fixed that issue so Travis CI is passing now.

**Comment by javierbertoli** [comment]
Thanks, @noelmcloughlin!!

**Pull request merged**